pkg/storage/parameterset_provider_helpers.go

```diff
@@ -56,26 +56,29 @@ func (p TestParameterSetProvider) Load(path string) (ParameterSet, error) {
 	fs := aferox.NewAferox(".", afero.NewOsFs())
 	var pset ParameterSet
 	err := encoding.UnmarshalFile(fs, path, &pset)
+	if err != nil {
+		return pset, fmt.Errorf("error reading %s as a parameter set: %w", path, err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error reading %s as a parameter set: %w", path, err)
```
We probably want that too.

Reply:
not possible